sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -460,9 +464,10 @@ void gnrc_rpl_send_DIS(gnrc_rpl_instance_t *inst, ipv6_addr_t *destination,
         for (size_t i = 0; i < num_opts; ++i) {
             if (options[i]->type == GNRC_RPL_OPT_SOLICITED_INFO) {
                 if ((pkt = _dis_solicited_opt_build(pkt,
-                    (gnrc_rpl_internal_opt_dis_solicited_t*)options[i])) == NULL) {
-                        return;
-                    }
+                                                    (gnrc_rpl_internal_opt_dis_solicited_t *)options
```

Comment:
I dislike the level of indent, but that would be for a discussion on the uncrustify settings, not this particular commit.